.github/workflows/continuous-integration.yml

```
@@ -59,6 +59,15 @@ jobs:
             dependencies: "highest"
             php-version: "8.4"
 
+          - symfony-require: "7.4.*"
```

```suggestion

          # Test LTS
          - symfony-require: "7.4.*"
```

Old comment should be renamed to `Test legacy LTS`